plugin-packs/postcss-preset-env/src/types/postcss-logical/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'postcss-logical' {
+	/** postcss-logical plugin options */
+	export type pluginOptions = {
+		/** Preserve the original notation. default: false */
+		preserve?: boolean,
+		/** Assume a direction for the document. default: null */
+		dir?: 'ltr' | 'rtl'
+	};
+}
```

Comment:
I don't want to convert logical to typescript now when we plan to rebuild it completely.